src/DynamoDbCache.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace pixelandtonic\dynamodb;
+
+use Aws\DynamoDb\Exception\DynamoDbException;
+use Yii;
+use yii\base\InvalidConfigException;
+use yii\caching\Cache;
+use yii\di\Instance;
+
+/**
+ * @property mixed $value
+ */
+class DynamoDbCache extends Cache
+{
+    public DynamoDbConnection|string|array $dynamoDb = 'dynamoDb';
```

Comment:
Is there a reason this is 3 different types? Seems like `DynamoDbConnection` would be the optimal choice?

Reply:
This is a Yii-ism, eg https://github.com/yiisoft/yii2-redis/blob/master/src/Cache.php#L110

This means it can grab it from the existing component `dynamoDb`, a specified `DynamoDbConnection`, or an array to pass to `Yii::createObject`.

Reply:
That rings a sore bell... ok I'll dismiss my comment.